src/service/actionssdk/conversation/conversation.ts

```diff
@@ -90,7 +91,8 @@ export interface ConversationResponse {
   richResponse: Api.GoogleActionsV2RichResponse
   expectUserResponse: boolean
   userStorage: string
-  expectedIntent?: Api.GoogleActionsV2ExpectedIntent
+  expectedIntent?: Api.GoogleActionsV2ExpectedIntent,
+  noInputPrompts?: Api.GoogleActionsV2SimpleResponse[],
```

Comment:
drop the commas here to match the rest of the code style

Reply:
Oh, sorry that.